packages/queries/src/query.ts

```diff
@@ -332,8 +333,12 @@ export function query<DB>(schema: DbSchema<DB>) {
         from<TTable extends Table<DB>>(table: TTable) {
             type TKeys = Array<Column<DB, TTable>>;
 
+            type TPrimaryKey = Array<string>;
```

Comment:
you don't need to declare this type

Reply:
Yes, but the type being passed to `QueryBuilder` for `groupingId` (before my changes) was `['id']`, and not `string[]`. Still need the type passed to `QueryBuilder`, but I can pass it directly.

Reply:
ah gotcha. Ok, just passing `string[]` is enough anyway. Alternatively you can probably do `typeof groupingId`